tests/unittests/OperatorTest.cpp

```diff
@@ -69,6 +69,262 @@ static Tensor createTensorConditionallyQuantized(ElemKind T,
   return isQuantizedElemKind(T) ? Tensor(T, dims, 1.0, 0) : Tensor(T, dims);
 }
 
+template <typename DataType>
+glow::Handle<bool>
+lessHelper(glow::PlaceholderBindings &bindings, glow::Module &mod,
+           glow::Function *F, glow::ExecutionEngine &EE, ElemKind DTy,
+           llvm::ArrayRef<DataType> xValues, llvm::ArrayRef<DataType> yValues,
+           llvm::ArrayRef<size_t> xDims, llvm::ArrayRef<size_t> yDims) {
+  auto *X = createPlaceholderConditionallyQuantized(mod, DTy, xDims, "X",
+                                                    DTy != ElemKind::FloatTy);
+
+  auto *Y = createPlaceholderConditionallyQuantized(mod, DTy, yDims, "Y",
+                                                    DTy != ElemKind::FloatTy);
+
+  bindings.allocate(llvm::dyn_cast<Placeholder>(X))->getHandle<DataType>() =
+      xValues;
+
+  bindings.allocate(llvm::dyn_cast<Placeholder>(Y))->getHandle<DataType>() =
+      yValues;
+
+  auto *cmpr = F->createNodeWithBroadcast<CmpLTNode>("cmpLT", -1, X, Y);
+
+  auto *save = F->createSave("save", cmpr);
+  auto *saveAlloc = bindings.allocate(save->getPlaceholder());
+
+  EE.compile(CompilationMode::Infer);
+  EE.run(bindings);
+
+  return saveAlloc->getHandle<bool>();
+}
+
+TEST_P(OperatorTest, less_int8) {
+  ENABLED_BACKENDS(Interpreter, CPU);
+
+  llvm::SmallVector<int8_t, 16> xValues = {
```

Comment:
Can we just use normal arrays here/in all the tests? I don't think we need a vector as we know the size statically.